docs/content/doc/usage/labels.en-us.md

```diff
@@ -0,0 +1,44 @@
+---
+date: "2023-03-04T19:00:00+00:00"
+title: "Usage: Labels"
+slug: "labels"
+weight: 13
+toc: false
+draft: false
+menu:
+  sidebar:
+    parent: "usage"
+    name: "Labels"
+    weight: 13
+    identifier: "labels"
+---
+
+# Labels
+
+You can use labels to classify issues and pull requests and to improve your overview over them.
+
+## Creating Labels
+
+Labels can be created for a repository by going to `Issues` and clicking on `Labels`. \
+For organizations, you can define organization-wide labels that are shared with all organization repositories, including both already-existing repositories as well as newly created ones. \
+Organization-wide labels can be created in the organization `Settings`.
+
+Labels have a mandatory name, a mandatory color, an optional description, and must either be exclusive or not (see `Scoped labels` below).
+
+When you create a repo, you can ensure certain labels exist by using the `Issue Labels` option. \
+This option lists a number of available label sets that are [configured globally on your instance](../customizing-gitea/#labels). \
+Its contained labels will all be created as well while creating the repo.
+
+## Scoped Labels
+
+You can decrease your error susceptibility by using scoped labels. \
+A scoped label is a label containing `/` in its name (not at either end of the name) and marking the label as `exclusive`. \
+For example, if label `A` is called `scope/item`, label `B` is called `scope/second-item`, and both are marked as `exclusive`, an issue cannot be labeled with both `A` and `B` at the same time. \
+Such an issue can have at most one of these labels, or none. \
+The scope of a label is determined based on the **last** `/`, so for example the scope of label `subscope/subscope2/item` would be `subscope/subscope2`.
```

Comment:
```suggestion
You can decrease your error susceptibility by using scoped labels. \
A scoped label is a label that is marked as `exclusive` and contains `/` in its name (not at either end of the name). \
For example, if label `A` is called `scope/item`, label `B` is called `scope/second-item`, and both are marked as `exclusive`, an issue cannot be labeled with both `A` and `B` at the same time. \
Issues can have at most one of these labels per scope, or none. \
The scope of a label is determined based on the **last** `/`, so for example the scope of label `subscope/subscope2/item` would be `subscope/subscope2`.
```
Unfortunately, you still had the unedited version.

Reply:
After #23164 I would consider any label with `/` in the name a scoped label, and then exclusive scoped labels are an additional option. So if you don't mind I will tweak the text to explain that.

Reply:
Yes, of course.
Please go ahead.